docker.py

```diff
@@ -959,6 +973,10 @@ def get_scipy_choices():
     cusparselt_ver='0.0.1',
 )
 
+codes['cusparselt-0.1.0'] = cusparselt_base.format(
```

Comment:
Could you check Jenkins failure and flake8 error as well?
I think the key names must be `cusparselt0.0.1-cudaXXX`? Same for `codes['cusparselt-0.0.1']`